src/services/user/hooks/admin.js

```diff
@@ -1,9 +1,21 @@
 const { authenticate } = require('@feathersjs/authentication').hooks;
+const { Forbidden } = require('@feathersjs/errors');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module '@feathersjs/errors'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4591179)